scripts/evaluate.py

```diff
@@ -124,21 +144,31 @@ def import_mobilenetv3_ssdlite(self, target="llvm", dtype="float32"):
         return (mod, params, input_shape, dtype, target)
 
     def import_deeplabv3(self, target="llvm", dtype="float32"):
-        import onnx
-
-        graph_file = os.path.abspath(
+        import tensorflow as tf
+        try:
+            tf_compat_v1 = tf.compat.v1
+        except ImportError:
+            tf_compat_v1 = tf
+        # Tensorflow utility functions
+        import tvm.relay.testing.tf as tf_testing
+
+        model_path = os.path.abspath(
             os.path.dirname(os.path.realpath(__file__))
```

Comment:
If possible, instead of replacing .onnx variant of deeplab with .pb, we could have kept it both.

Reply:
Done